### DISCUSSION TITLE
Bottle for notmuch does not include a standard part of package (the script notmuch-git/nmbug)

### DISCUSSION THREAD

**Discussion by hgvhgv** [Homebrew/discussions#5204] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.12-59-ga3e5e3f
ORIGIN: https://github.com/Homebrew/brew
HEAD: a3e5e3f7a0e98fe8ffbd8894a9ea0c1fb5c9884f
Last commit: 8 hours ago
Core tap JSON: 14 Mar 22:01 UTC
Core cask tap JSON: 14 Mar 22:01 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_BROWSER: /Applications/Brave Browser.app
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6.3-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Warning: Putting non-prefixed coreutils in your path can cause GMP builds to fail.
```

### Description of issue

The bottle for notmuch (https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fnotmuch formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/n/notmuch.rb) does not include notmuch-git, which is a python script (formerly and also called nmbug) included in the upstream package and installed by default. Is there a reason? I installed the script myself, but using it requires that notmuch be built with the sfsexp library ("small fast s-expression"; https://github.com/mjsottile/sfsexp). I can install this library and build notmuch myself, but I'd rather have my notmuch install managed by homebrew. What would it take to build the brew bottle of notmuch with sfsexp and the notmuch-git script?

**Comment by carlocab** [comment, marked as answer]
It looks like we would need (1) an `sfexp` formula and (2) for `sfexp` to be added as a dependency of `notmuch`.

I don't think any maintainer uses `notmuch` regularly, so I think it'd be best if you opened PRs for these yourself so that you can verify that they work as expected. I suggest starting with a single PR that adds `sfexp` first.

Here are some docs to help you get started:

https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request
https://docs.brew.sh/Formula-Cookbook

Feel free to ask if you have any questions or get stuck.

**Comment by hgvhgv** [comment]
To close this out:`sfsexp` was added to homebrew and homebrew now installs `notmuch-git` with `notmuch`
